packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-chip-styles.js

```diff
@@ -22,6 +22,10 @@ const chip = css`
     cursor: var(--lumo-clickable-cursor);
   }
 
+  :host([focused]) {
+    background-color: var(--lumo-contrast-40pct);
```

Comment:
No-focus ring style like other keyboard interactive components?

Reply:
The whole idea behind this feature (focusing chips) is that they can be only focused with keyboard, so having an extra attribute just to be aligned with other components seems redundant to me.

Reply:
Oh sorry, I don't mean the attribute - I was missing the blue border :)

Reply:
I’m hesitant in adding that additional outline as well. In the case of something like Date Picker, where the field has it and other components within the overlay have it as well feels less problematic as they are not nested. But here they would be, making it look odd.

Reply:
Updated to use the primary color.